html/semantics/scripting-1/the-script-element/microtasks/evaluation-order-1-nothrow-static-import.any.js

```diff
@@ -1,5 +1,5 @@
-// META: global=dedicatedworker-module,sharedworker-module
-// META: script=./resources/evaluation-order-setup.js
+// META: global=dedicatedworker-module,sharedworker-module,shadowrealm
+// META: script=/html/semantics/scripting-1/the-script-element/microtasks/resources/evaluation-order-setup.js
```

Comment:
Oh, this is annoying. Do we need to do this anywhere else? If so I'd try to adapt the serve.py script to resolve relative URLs

Reply:
Since this was also raised in #49286, I'll try to tackle that next.

Reply:
→#49386